test/it/uk/gov/hmrc/exports/services/mapping/ExportsPointerToWCOPointerISpec.scala

```diff
@@ -0,0 +1,25 @@
+package uk.gov.hmrc.exports.services.mapping
+
+import org.scalatest.matchers.must.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+
+class ExportsPointerToWCOPointerISpec extends AnyWordSpec with Matchers {
+
+  "ExportsPointerToWCOPointer.getWCOPointers" should {
+
+    "return the expected WCO Pointers for the provided Exports Pointers" in {
+      val dynamicExportsPointer = "declaration.items.$98.additionalFiscalReferencesData.references.$43.country"
```

Comment:
This is not how the ExportsField pointer will look. It would be 
`"declaration.items.98.additionalFiscalReferencesData.references.43.country"` 
No '$' symbol.